config/server.webpack.config.babel.js

```diff
@@ -0,0 +1,5 @@
+/*
+ * Common config for server and server tests
```

Comment:
Also I think the comments should probably address the `why`, more than the `how` and `what`.